test/shared/src/test/scala/monocle/TestInstances.scala

```diff
@@ -78,11 +77,6 @@ trait TestInstances extends PlatformSpecificTestInstances with cats.instances.Al
         .perturb(seed, (t.head, t.tail.value))
     )
 
-  implicit def uuidArbitrary: Arbitrary[UUID] = Arbitrary(UUID.randomUUID)
-
-  implicit def uuidCoGen: Cogen[UUID] =
-    Cogen[(Long, Long)].contramap[UUID]((u: UUID) => (u.getMostSignificantBits, u.getLeastSignificantBits))
```

Comment:
ScalaCheck already provides these. That implementation is preferable because:
1. It respects the ScalaCheck seed so it is reproducible.
2. `randomUUID` was recently affected by a CVE. See https://github.com/scala-js/scala-js/security/advisories/GHSA-j2f9-w8wh-9ww4